### PR TITLE
ISPN-5447 fixed spring schema usage in tests

### PR DESCRIPTION
**Pull request by michalszynkiewicz**
1. infinispan-spring schemas renamed to 8.2 (the number was used in spring4 pom.xml)
2. spring.schemas now pointing to proper version of schema

### CODE REVIEW

File: spring/spring/pom.xml [hunk 1]
```diff
@@ -237,6 +237,43 @@
                     </instructions>
                 </configuration>
             </plugin>
+            <plugin>
```
Comment (slaskawi) [inline spring/spring/pom.xml]: Currently we support both Spring 3 (placed in this module) and Spring 4 (placed here https://github.com/infinispan/infinispan/tree/master/spring/spring4). Please add similar changes to Spring 4. Spring 3 support will be dropped soon.

Comment (slaskawi) [inline spring/spring/pom.xml]: Clarified... Spring 4 works a bit differently because of Spring commons package. Everything is fine.

Comment (michalszynkiewicz) [inline spring/spring/pom.xml]: spring 4 works differently. XSD is in spring4-common, while tests are in spring4-embedded.
In such situation there's no problem with classpath resolution of the XSD.

Comment (michalszynkiewicz) [inline spring/spring/pom.xml]: this turns out to be unnecessary after changing xsd version to 8.2

I will remove changes in pom.xml.

**Comment by slaskawi** [comment]
:+1: Looks good. Let's wait for CI

**Comment by tristantarrant** [comment]
Pushed, thanks

**Pull request closed without merging**